common/constants.js

```diff
@@ -105,7 +105,7 @@ module.exports = Object.freeze({
     SHUTDOWN_WAIT_TIME: 5000
   },
   BACKUP: {
-    BACKUP_START_TIMEOUT: 15, //SECONDS
+    BACKUP_START_TIMEOUT: 60, //SECONDS
```

Comment:
Any reason for bumping this and keeping it at 60? its always better to put time units as part of variable name as it makes it readable at all places as in `BACKUP_START_TIMEOUT_IN_SECS`

Reply:
Yes in azure it takes 32 seconds for the backupService to start and update the apiserver.

Reply:
Renamed